pkg/gitreceive/build.go

```diff
@@ -112,6 +110,20 @@ func build(
 		}
 	}
 
+	_, disableCaching := appConf.Values["DEIS_DISABLE_CACHE"]
```

Comment:
this would mean user has to do config:set/unset DEIS_DISABLE_CACHE to disable/enable the cache.
Can we check the value instead?